lib/linguist/languages.yml

```diff
@@ -1138,6 +1138,16 @@ EJS:
   tm_scope: text.html.js
   ace_mode: ejs
   language_id: 95
+EML:
+  type: data
```

Comment:
Should the type be changed to `prose` ?

Reply:
No, better to leave it as `data`. Most fields in an average e-mail are generated by mail servers, with the only real human-written field being the body and subject lines. `prose` is more for lightweight, hand-typed messages that require little or no processing to be read comfortably.